modules/gdscript/gdscript_warning.cpp

```diff
@@ -40,10 +40,10 @@ String GDScriptWarning::get_message() const {
 	switch (code) {
 		case UNASSIGNED_VARIABLE:
 			CHECK_SYMBOLS(1);
-			return vformat(R"(The variable "%s" was used before being assigned a value.)", symbols[0]);
+			return vformat(R"(The variable "%s" is being used before being assigned a value.)", symbols[0]);
```

Comment:
I am not a native speaker, but the double "being" looks a bit cumbersome to me. In my opinion, "was" or "is" looks better. Every use of an unassigned variable is an accomplished fact that was detected by the analyzer, and not an ongoing "real" event/process. But we should choose which language we want to use, more technical or more user-friendly.

Reply:
I agree, and I noticed it also wasn't consistent with some of the other messages